.github/workflows/python.yml

```diff
@@ -254,8 +255,54 @@ jobs:
           name: wheels
           path: dist
 
+      # All platforms are currently creating the same rrd file, upload one of them
+      - name: Save RRD artifact
+        if: matrix.platform == 'linux'
+        uses: actions/upload-artifact@v3
+        with:
+          name: rrd
+          path: rrd
+
   # ---------------------------------------------------------------------------
 
+  upload_rrd:
+    name: Upload RRD to GCloud
+    permissions:
+      contents: "read"
+      id-token: "write"
+    needs: [wheels]
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: Download Artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: rrd
+          path: rrd
+
+      - id: "auth"
+        uses: google-github-actions/auth@v1
+        with:
+          workload_identity_provider: ${{ secrets.GOOGLE_WORKLOAD_IDENTITY_PROVIDER }}
+          service_account: ${{ secrets.GOOGLE_SERVICE_ACCOUNT }}
+
+      - name: Add SHORT_SHA env property with commit short sha
+        run: echo "SHORT_SHA=`echo ${{github.sha}} | cut -c1-8`" >> $GITHUB_ENV
+
+      - name: "Upload RRD"
+        uses: google-github-actions/upload-cloud-storage@v1
+        with:
+          path: "rrd"
+          destination: "rerun-example-rrd/commit/${{env.SHORT_SHA}}"
+          parent: false
+
+      - name: "Upload RRD (tagged)"
+        if: startsWith(github.ref, 'refs/tags/v')
```

Comment:
Maybe we should do `if: startsWith(github.ref, 'refs/tags/v') || github.ref == 'refs/tags/latest'` here and elsewhere to always get a `latest` web viewer automatically

Reply:
Generally agree but I'm going to handle the `latest` / `prerelease` logic a bit differently in another PR. There's a bit of a mess around what does / doesn't trigger a workflow. `github.ref` doesn't mean "this commit happens to be tagged with latest" but rather "pushing the tag latest is what triggered this PR." But running the whole pipeline a second time (including wheel building on all platforms) seems heavy-handed to effectively update a pointer.